Fix error in build pipeline

```
Run $(npm bin)/nyc --reporter=cobertura npm test
[6](https://github.com/center-for-threat-informed-defense/attack-workbench-rest-api/actions/runs/4327535183/jobs/7556278699#step:7:7)
/home/runner/work/_temp/1d5b2f9f-f3f2-4c2f-b3aa-8999cd2a2e29.sh: line 1: Unknown: command not found
[7](https://github.com/center-for-threat-informed-defense/attack-workbench-rest-api/actions/runs/4327535183/jobs/7556278699#step:7:8)
Error: Process completed with exit code 127.
```

--- a/.github/workflows/ci-workflow.yml
+++ b/.github/workflows/ci-workflow.yml
@@ -17,7 +17,7 @@ jobs:
       - run: npm install
       - run: npm run lint
       - run: npm run snyk
-      - run: $(npm bin)/nyc --reporter=cobertura npm test
+      - run: npx nyc --reporter=cobertura npm test
       - name: Upload Coverage to CodeCov
         uses: codecov/codecov-action@v2
         with:

--- a/.github/workflows/ci-workflow.yml
+++ b/.github/workflows/ci-workflow.yml
@@ -4,6 +4,7 @@ on:
   push:
     branches: [master, develop]
   pull_request:
+  workflow_dispatch:
 
 jobs:
   build:

